sphinx/v1.0-0/index.html

```diff
@@ -8,7 +8,7 @@
     
     <title>DS3 Python SDK &mdash; Ds3 Python SDK 0 documentation</title>
     
-    <link rel="stylesheet" href="_static/classic.css" type="text/css" />
+    <link rel="stylesheet" href="_static/default.css" type="text/css" />
```

Comment:
classic.css is the css file in source, so we probably don't want to change this.